h/static/scripts/widget-controller.js

```diff
@@ -334,29 +347,33 @@ module.exports = function WidgetController(
   };
 
   /**
-   * Return the offset between the top of the window and the top of the
-   * first annotation card.
+   * Return the vertical scroll offset for the document in order to position the
+   * annotation thread with a given `id` or $$tag at the top-left corner
+   * of the view.
    */
-  function cardListYOffset() {
-    var cardListTopEl = document.querySelector('.js-thread-list-top');
```

Comment:
This class remains on the spacer in the template but is now not referenced anywhere.